pom.xml

```diff
@@ -315,6 +317,34 @@
         <version.spotless-maven-plugin>2.45.0</version.spotless-maven-plugin>
         <version.maven-antrun-plugin>3.1.0</version.maven-antrun-plugin>
 
+        <!-- >>> Version of dependencies that we need to keep in the root to share between build and "bom" pom files -->
```

Comment:
Question: if we import the platform BOM into the skinny BOM, would the "flattening" be able to only keep the dependencies explicitly referenced in the skinny BOM, applying versions coming from the platform BOM?